p2p/pre2p/network.go

```diff
@@ -0,0 +1,100 @@
+package pre2p
```

Comment:
Again, might be out of scope - but pre2p needs documentation justifying why there's little to no unit tests and how the e2e covers